README.md

```diff
@@ -49,6 +49,65 @@ const server = new InversifyExpressServer(container, null);
 })
 ```
 
+## What is SiberMetrics and how it works.
```

Comment:
It's not quite a "histogram of an HTTP request". 
https://prometheus.io/docs/introduction/overview/